paths.pegjs

```diff
@@ -79,14 +79,20 @@ CqExpression = operation:("cq") _ "(" _ path:Expression _ ")" {
   }
 }
 
+SplitExpression = operation:("split") _ "(" _ path:Expression _ ")" {
```

Comment:
shouldn't this be something like 
```
SplitExpression = operation:("split") _ "(" _ path:Expression _ "," path:SplitBy ")" {
```
or something?

Reply:
Ideally, it would be something like 

```
SplitExpression = operation:("split") _ "( _ path:Expression _ separator:("," _ StringLiteral)? _ ") {
```

But the path always gobbles up any separator.